crates/duty-tracker/src/contract_manager.rs

```diff
@@ -398,51 +382,30 @@ impl ContractManager {
                             // this could be a transient issue, so no need to break immediately
                         }
                     },
+                    instant = interval.tick() => {
+                        debug!(?instant, "constructing nags");
 
-                    // Finally we pull from the duties channel and execute those.
-                    Some((contract_txid, duty_result)) = duty_response_receiver.recv() => {
-                        match duty_result {
-                            Ok(contract_duties) if !contract_duties.is_empty() => {
-                                info!(%contract_txid, num_duties=%contract_duties.len(), "received duties from contract actor");
-                                duties.extend(contract_duties);
-                            },
-                            Ok(_) => {
-                                trace!(%contract_txid, "contract actor processed event with no duties");
-                            },
-                            Err(e) => {
-                                error!(%contract_txid, %e, "contract actor failed to process event");
-                                // For aggregate signatures, this is catastrophic
-                                // Check if this was an aggregate signature event and break if so
-                                // For now, we'll log the error and continue
-                            }
-                        }
-                    },
-
-                    _instant = interval.tick() => {
-                        info!("nagging peers for necessary p2p messages");
```

Comment:
don't remove this log

Reply:
It is not removed, check above it is there as a `debug!`:

```rs
debug!(?instant, "constructing nags");
```